scintillometry/tests/test_io_psrfits_setter.py

```diff
@@ -0,0 +1,70 @@
+# Licensed under the GPLv3 - see LICENSE
```

Comment:
Presumably, writing will be tested here too, so maybe `test_io_psrfits_writer.py`

Reply:
This will be done for the next several PR

Reply:
OK, just the file name change for now then!